content/en/apps/guides/messaging/rapidpro.md

```diff
@@ -0,0 +1,114 @@
+---
+title: RapidPro SMS Gateway
+linkTitle: "RapidPro"
+weight:
+description: >
+    Integration for sending and receiving SMS
+relatedContent: >
+    apps/reference/app-settings/sms
+    apps/guides/forms/app-form-sms
+    apps/guides/messaging/sms-states
+    apps/guides/messaging/message-loops
+    apps/guides/messaging/shortcodes
+---
+
+
+As of v3.11.0, SMS messages can be sent and received using [RapidPro]({{% ref "apps/features/integrations/rapidpro" %}}) as an SMS Gateway. 
```

Comment:
It's probably best to be more generic about the message type, since outgoing messages could be sent via channels other than SMS (eg Telegram).

```suggestion
As of v3.11.0, messages can be sent and received using [RapidPro]({{% ref "apps/features/integrations/rapidpro" %}}) as an messaging gateway. 
```